sunbeam-python/sunbeam/features/caas/etc/caas-setup/main.tf

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 terraform {
-  required_version = ">= 0.15.1"
+  required_version = ">= 0.17.0"
```

Comment:
This is supposed to be terraform version. Can you update it to version used in snapcraft.yaml. (Similarly in cloud/etc/demo-setup)
Shall we consider bumping terraform version as well as subsequent PR.